src/components/studentDetailCards/studentDataSection/StudentDataCard.tsx

```diff
@@ -34,8 +34,8 @@ const StudentDataCard: React.FC = () => {
 
   return (
     <div data-testid="StudentDataCard">
-      {aboutData.map((studentData) => (
-        <div key={studentData.id} className="flex flex-col gap-4">
+      {aboutData.map((studentData, index) => (
+        <div key={studentData.id || index} className="flex flex-col gap-4">
```

Comment:
Fallback to a toast when this or other error occurs. Dismiss the PR and make an add toast new PR